lib/template-mixins.js

```diff
@@ -514,6 +514,14 @@ module.exports = function (options, deprecated) {
             };
         };
 
+        res.locals.renderField = function () {
+            return function () {
+                var mixin = this.mixin;
```

Comment:
I'd be tempted to take this opportunity to apply a default mixin of `input-text` to avoid having to manually specify a `mixin` property on text fields when creating forms.